src/renderer/styles/LargeButton.scss

```diff
@@ -0,0 +1,23 @@
+@import 'color';
+@import 'button-state';
+
+$button-width: 24px;
+$button-height: 100px;
+
+.button {
+  width: $button-width;
+  height: $button-height;
+  border-radius: 5px;
```

Comment:
can we go with 4px here? I _think_ we use that for the Button component too so it'd be nice to be consistent